services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1474,7 +1480,52 @@ public void sendPortUpEvents(final IOFSwitch sw) throws SwitchOperationException
                 }
             }
         }
+    }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PortConfigurationResponse configurePort(final PortConfigurationRequest request) 
```

Comment:
I don't see the code which applies the provided speed. Was it implemented? If not, let's add a todo comment here or remove the speed field from PortConfigurationRequest.

Reply:
Added TODO Comment inside the method where the actual code is expected

Reply:
addressed